main

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+require './app'
+
```

Comment:
Please fix the linter checks as all the checks are failing on PR.

Reply:
Hello @SajjadAhmad14,
I am on Windows and this bash command is not working.